tests/blobbasefee_test.go

```diff
@@ -91,3 +66,34 @@ func getBlobBaseFeeFrom(header *types.Header) uint64 {
 	}
 	return eip4844.CalcBlobFee(excessBlobGas).Uint64()
 }
+
+func TestBlobBaseFee_CanReadBlobGasUsed(t *testing.T) {
+	require := require.New(t)
+	net, err := StartIntegrationTestNet(t.TempDir())
+	require.NoError(err, "Failed to start the fake network: ", err)
+	defer net.Stop()
+
+	client, err := net.GetClient()
+	require.NoError(err, "failed to get client; ", err)
+	defer client.Close()
+
+	// Get blob gas used from the block header of the latest block.
+	block, err := client.BlockByNumber(context.Background(), nil)
+	require.NoError(err, "failed to get block header; ", err)
+	require.Empty(*block.BlobGasUsed(), "unexpected value in blob gas used")
```

Comment:
since I'm dereferencing the result of `BlobGasUsed`, the type of that value is uint64, for that type, `require.Empty` checks that it has value zero. Or at least that is what I understood from the documentation.